### PR TITLE
Fixing a message or two + add test/comparisonRules.json to .gitignore (for when it's in)

### PR DESCRIPTION
**Pull request by brentlintner**

### CODE REVIEW

File: stressmodel.json [hunk 1]
```diff
@@ -519,7 +519,7 @@
             "present": false
           }
         ],
-        "message": "Assigned LTS 0 since footway is a crossing without bicycle access.",
+        "message": "Assigned LTS -1 since footway is a crossing without bicycle access.",
```
Comment (JacksonZeitspace) [inline stressmodel.json:522]: I think this one should be 0

**Comment by brentlintner** [comment]
Closing this since it might be the opposite way we want to fix this.

File: stressmodel.json [hunk 2]
```diff
@@ -358,7 +358,7 @@
             "present": false
           }
         ],
-        "message": "Assigned LTS 0 since access=no or private and no bicycle=yes or designated tag.",
+        "message": "Assigned LTS -1 since access=no or private and no bicycle=yes or designated tag.",
```
Comment (JacksonZeitspace) [inline stressmodel.json:361]: This one might actually be -1, not 100% sure

**Review by JacksonZeitspace — CHANGES_REQUESTED** [review]

File: stressmodel.json (hunk 2 above)
Comment (JacksonZeitspace) [reply stressmodel.json:361]: Looking at the spreadsheet, it looks like a 0 too?

**Pull request closed without merging**